workflow/drilling.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Drilling Into Your Data
+---
+
+The Drill Down feature enhances your ability to investigate, analyze, and resolve issues without needing to switch contexts. Instead of sifting through large volumes of data indiscriminately, focus your efforts on drilling into relevant and focused subsets of data in order to save your team time and effort.
+
+This feature is available across all of Middleware and works by passing the context of your data from everything to everything. Clicking on any of Middleware's interactive tables will produce popouts that divulge the interconnectedness of your system infrastructure in a single window.
```

Comment:
Let's change "from everything to everything" to "passing context through from the beginning to end of a request."

Reply:
Also "...Middleware's interactive tables will produce popouts that allow you to drill to root cause regardless of which direction you start from."